serialization/journal.proto

```diff
@@ -168,6 +168,37 @@ message CelestialFromParent {
   optional Return return = 3;
 }
 
+message CelestialRotation {
```

Comment:
It would make me happier if we could find a systematic naming convention where all related methods start with the same word, and we have an `interface_foo.cpp` to hold their implementation.  `interface.cpp` is just too messy at this point.

Now we have to argue about the names.  `SetMainBody` is a bit special, so let's keep it as it is (and in `interface.cpp`).  All the other methods have `rotation` in their name.  A suggestion:
```
RotationOfCelestial
RotationOfCelestialSphere
RotationGetInitialInDegrees
RotationGetPeriod
```